content/references/rippled-api/api-conventions/basic-data-types.md

```diff
@@ -81,76 +81,27 @@ If you do not specify a ledger, the `current` (in-progress) ledger is chosen by
 **Note:** Do not rely on this default behavior for specifying a ledger; it is subject to change. Always specify a ledger version in the request if you can.
 
 
-## Currencies
+## Specifying Currency Amounts
 
-There are two kinds of currencies in the XRP Ledger: XRP, and everything else. There are many differences between the two:
+There are two kinds of currencies in the XRP Ledger: XRP, and issued currencies. These two types of currencies are specified in different formats, and they have different amounts of precision and different rounding behavior.
```

Comment:
I simplified this sentence to just:

> These two types of currencies are specified in different formats, with different precision and rounding behavior.

That's probably for the best since the discussion of precision in this case is not strictly a question of "more" or "less".